app/cinema/bar.py

```diff
@@ -0,0 +1,7 @@
+from app.people.customer import Customer
+
+
+class CinemaBar:
+    @staticmethod
+    def sell_product(customer: "Customer", product: str) -> None:

```

Comment:
You should provide type hint without `"`
```suggestion
    def sell_product(customer: Customer, product: str) -> None:

```